outputs.tf

```diff
@@ -0,0 +1,27 @@
+output "detector" {
+  description = "GuardDuty filter"
+  value       = aws_guardduty_detector.this
+}
+output "filter" {
+  description = "GuardDuty filter"
+  value       = var.filter != null ? aws_guardduty_filter.this : null
+}
+
+output "ipset" {
+  description = "GuardDuty ipset"
+  value       = var.ipset != null ? aws_guardduty_ipset.this : null
+}
+
+output "threatintelset" {
+  description = "GuardDuty threatintelset"
+  value       = var.threatintelset != null ? aws_guardduty_threatintelset.this : null
```

Comment:
```suggestion
  value       = aws_guardduty_threatintelset.this
```

Reply:
Good catch.  I missed this when i updated the threatintelset default from null to an empty list.